python/requirements-wheel-build.txt

```diff
@@ -1,3 +1,4 @@
+build
```

Comment:
Can you try to remove `wheel` below? I don't think it's gonna be necessary anymore.

Reply:
And, actually, you might perhaps be able to remove this entire file, since all dependencies here are also listed in `pyproject.toml`.

Reply:
(and then the same may also apply to `requirements-build.txt`)

Reply:
Another thing: the `python_version<'3.9'` part can be removed since 3.9 is the minimum supported Python version :)